lib/search/query_components/booster.rb

```diff
@@ -23,10 +23,24 @@ def wrap(core_query)
   private
 
     def boost_filters
-      format_boosts + [time_boost, closed_org_boost, devolved_org_boost, historic_edition_boost]
+      boosts = format_boosts + [time_boost, closed_org_boost, devolved_org_boost, historic_edition_boost]
+
+      if @search_params.format_boosting_b_variant?
+        boosts + [guidance_boost]
+      else
+        boosts
+      end
     end
 
     def boosted_formats
+      if @search_params.format_boosting_b_variant?
+        FORMAT_BOOST_CONFIG["format_boosts"]
```

Comment:
I think it might be clearer to inline `formats_boosted_by_index` (or restructure it in some other way) - so that it's more obvious that `FORMAT_BOOST_CONFIG["format_boosts"]` is used in both cases, and the A case is just doing something extra.

As is, it's a bit hard to see the difference.